bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -157,6 +177,17 @@ public Empty mutateRow(MutateRowRequest request) throws ServiceException {
   }
 
   @Override
+  public MutateRowsResponse mutateRows(MutateRowsRequest request) throws ServiceException {
+    return clientCallService.blockingUnaryCall(createMutateRowsCall(request), request);
+  }
+
+  @Override
+  public ListenableFuture<MutateRowsResponse> mutateRowsAsync(MutateRowsRequest request) {
+    expandPoolIfNecessary(this.bigtableOptions.getChannelCount());
```

Comment:
In the implementation of expandPoolIfNecessary, should something more serious than LOG.info be used if the pool can't be resized?